### PR TITLE
[DOCKER] Support HTTPS as well by making SFTP optional

### PR DESCRIPTION
**Pull request by SerhatG**
We also want people to specify in their project settings whether they want to use SFTP or HTTPS.

### CODE REVIEW

File: docker/build-database.sh [hunk 1]
```diff
@@ -52,16 +51,16 @@ mkdir -p "${PGDATA}" && chown -R postgres:postgres "${PGDATA}" && chmod 777 "${P
 # create db-data folder for the repo
 mkdir -p "${DBDATA_PATH}"
 
-# configure repo with the readonly SFTP password given as argument
-echo "\$sftp_data_readonly_password = '${SFTP_READONLY_PASSWORD}'" >> "${DBCONFIG_PATH}/AeriusSettings.User.rb"
+# configure repo with the readonly SFTP password given as argument if set
+[[ -n "${SFTP_READONLY_PASSWORD}" ]] && echo "\$sftp_data_readonly_password = '${SFTP_READONLY_PASSWORD}'" >> "${DBCONFIG_PATH}/AeriusSettings.User.rb"
```
Comment (BertScholten) [inline docker/build-database.sh:55]:
```suggestion
[[ -n "${SFTP_READONLY_PASSWORD}" ]] && echo "\$sftp_data_readonly_password = '${SFTP_READONLY_PASSWORD}'" >> "${DBCONFIG_PATH}/AeriusSettings.User.rb"
[[ -n "${HTTPS_PASSWORD}" ]] && echo "\$https_data_password = '${HTTPS_PASSWORD}'" >> "${DBCONFIG_PATH}/AeriusSettings.User.rb"
```
Not sure if this the correct way to go if both are present, but the https password should be treated similar to the sftp one.

**Review by BertScholten — CHANGES_REQUESTED** [review]

File: docker/build-database.sh (hunk 1 above)
Comment (SerhatG) [reply docker/build-database.sh:55]: ah yeah, didn't think about adding the HTTPS credentials (yet).. Can do that directly..

**Comment by SerhatG** [comment]
I took the liberty to add `username` as well, as we are going to use accounts per person instead of one generic account. (in case developers want to use this locally - even though I can technically provide it in the User settings one)

**Review by BertScholten — APPROVED** [review]
LGTM

**Pull request merged**